src/overdrive.js

```diff
@@ -4,6 +4,19 @@ import ReactDOM  from 'react-dom'
 const renderSubtreeIntoContainer = ReactDOM.unstable_renderSubtreeIntoContainer;
 const components = {};
 
+const prefix = styles => {
+    const propertiesToPrefix = ['transform', 'transformOrigin', 'transition'];
+    const prefix = 'Webkit';
+    const prefixedStyles = {};
+    Object.keys(styles).forEach(property => {
+        if (propertiesToPrefix.indexOf(property) >= 0) {
```

Comment:
Seems like `indexOf` might bring a performance hit. What do you think about `if (propertiesToPrefix === property) {`?